software/tubearchivist.yml

```diff
@@ -0,0 +1,18 @@
+name: "Tube Archivist"
+website_url: "https://tubearchivist.com/"
+source_code_url: "https://github.com/tubearchivist/tubearchivist"
+description: "Tube Archivist is a handy tool that helps you efficiently organize, search, and enjoy your growing YouTube video collection offline, using metadata indexing and a user-friendly web interface, allowing you to subscribe to channels, download videos and keep track of viewed content easily."
+licenses:
+  - GPL-3.0
+platforms:
+  - Python
+  - Javascript
+  - Django
+  - Docker
+tags:
+  - Media Streaming
+  - Media Streaming - Video Streaming
+  - Archiving and Digital Preservation (DP)
+  - Automation
```

Comment:
```suggestion
tags:
  - Media Streaming - Video Streaming
```

Reply:
Why no archiving? It's literally in the name.. Automation because it automatically syncs new videos.

Reply:
> Why no archiving? It's literally in the name

Mis-click, of course Archiving is valid. updated my suggestion

> Automation because it automatically syncs new videos.

Automation is for generic automation/workflow tools that do not fit properly in any other category. If we started going this way most software in the list would qualify for the Automation section.